src/pages/Login.js

```diff
@@ -0,0 +1,86 @@
+import {
+  AnimatePresence, AnimateSharedLayout, motion, useCycle,
+} from 'framer-motion';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import LoginInForm from '../component/myForm/LogInForm';
+import SignUpForm from '../component/myForm/SignUpForm';
+import PopUp from '../component/popups/PopUp';
+import Layout from '../component/wrapper/Layout';
+import styles from './pages.module.scss';
+
+const menuState = {
+  intro: {
+    goto: 'Already have an account?',
+  },
+  login: {
+    goto: 'Create account!',
+  },
+  signup: {
+    goto: 'Back',
+  },
+};
+
+const Login = () => {
+  const [toggle, setToggle] = useCycle('intro', 'login', 'signup');
+  const { status } = useSelector((state) => state.sessions);
+
+  return (
+    <Layout preAnimation="fadeIn">
+      <section
+        className={styles.loginMainContainer}
+        style={{
+          display: 'flex', placeContent: 'center', flexDirection: 'column', width: '100vw', alignItems: 'center',
+        }}
+      >
+        { status === 'error' && (
+          <PopUp message="Wrong email or password!" skip />
+        )}
+        <AnimateSharedLayout>
+          <motion.div
+            style={{
+              display: 'flex', justifyContent: 'center', alignItems: 'center', flexDirection: 'column',
+            }}
```

Comment:
- [ ] If you can use classes for styling here, it would be nice, since it's not a good practice to use the `style` attribute.